src/docs/development/platform-integration/platform-views.md

```diff
@@ -212,27 +260,28 @@ For more information, see the API docs for:
 [`PlatformViewRegistry`]: {{site.api}}/javadoc/io/flutter/plugin/platform/PlatformViewRegistry.html
 [`PlatformView`]: {{site.api}}/javadoc/io/flutter/plugin/platform/PlatformView.html
 
-Finally, modify your `build.gradle` file to require
-version 19 (or later) of the Android API.
+Finally, modify your `build.gradle` file to require either 
+the minimal Android SDK version:
 
 ```gradle
 android {
     defaultConfig {
-        minSdkVersion 19
+        minSdkVersion 19 // if using Hybrid composition.
+        minSdkVersion 20 // if using Virtual display.
     }
 }
 ```
 
 ### Performance
 
 Hybrid composition on Android works best on Android 10 or above.
-On lower versions of Android, the Flutter UI can appear slower
-while the native view is rendered on the screen.
-This overhead is due to the synchronization of Flutter frames to the
-Android view system.
+On lower versions of Android, the engine makes a device-host-device
```

Comment:
Can we spell this out in more detail? I'm not sure what device-host-device copy means?

Is it correct to say something like:
On Android versions older than 10, the hybrid composition mode copies each Flutter frame out of the graphic memory into main memory and then copied back to a GPU texture. As this copy happens per frame the performance of the entire Flutter UI may be impacted significantly.